image-layout.md

```diff
@@ -29,6 +29,11 @@ The image layout is as follows:
   - It MUST be an [image index](image-index.md) JSON object.
   - See [index.json](#indexjson-file) section
 
+**Implementor's Note:**
+Additional files may be included in the directory.
+Implementations should not error when encountering unknown files.
+A common usage includes the `manifest.json` file associated with a saved docker image.
```

Comment:
```suggestion
A common usage includes the `manifest.json` file associated with a saved Docker image.
```

We could add a reference for this file too, but I don't think that's actually important. 👍

Another alternative to resolve the proper noun in a different way:

```suggestion
A common usage includes the `manifest.json` file associated with the older `docker save` format (for backwards compatibility).
```

Reply:
The reference for `manifest.json` could be done in the Docker documentation since I believe they own that format.

I went towards option 2. I have a personal hesitancy to name companies in the spec, and I've always associated Docker (capital) with the company and docker (lower case) with the tool/CLI.

Reply:
Ironically, the best reference for it that I've found is the one @jonjohnsonjr wrote in https://github.com/google/go-containerregistry/tree/6bce25ecf0297c1aa9072bc665b5cf58d53e1c54/pkg/v1/tarball#structure :smile: